### PR TITLE
{bio}[intel/2019b] Flye v2.7 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3117.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/752c6fe5a55ec0cbab0a6a0840e96212 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2501.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/67c2bf58442df0a124c62c4fa0511b2e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/7dc6aaa6658c4670ea356a7a754b1305 for a full test report.

File: easybuild/easyconfigs/f/Flye/Flye-2.7-intel-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+name = 'Flye'
+version = '2.7'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/fenderglass/Flye'
+description = """Flye is a de novo assembler for long and noisy reads, such as those produced by PacBio
+ and Oxford Nanopore Technologies."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
```
Comment (Micket) [inline easybuild/easyconfigs/f/Flye/Flye-2.7-intel-2019b-Python-3.7.4.eb]: Doesn't look like there is any MPI or LAPACK used in this. Can it be dropped to iccifort?

**Comment by jfgrimm** [comment]
closing as outdated

**Pull request closed without merging**